src/mame/video/galaga.cpp

```diff
@@ -8,6 +8,8 @@
 
 ***************************************************************************/
 
+#include <stdlib.h>
```

Comment:
Please prefer C++ standard library over C runtime.  Also, don't `#include` things before the PCH.

Reply:
change committed